examples/forward/plot_forward_sensitivity_maps.py

```diff
@@ -28,7 +28,8 @@
 subjects_dir = data_path + '/subjects'
 
 # Read the forward solutions with surface orientation
-fwd = mne.read_forward_solution(fwd_fname, surf_ori=True)
+fwd = mne.read_forward_solution(fwd_fname)
+fwd = mne.convert_forward_solution(fwd, surf_ori=True)
```

Comment:
you need `use_cps=False` to avoid a warning (and for future compat) here, right?